com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderProgramBuilder.java

```diff
@@ -526,7 +564,7 @@ public void BuildShader(String[] args, ES2ToES3Converter.ShaderType shaderType)
            is.read(inBytes);
            ByteArrayInputStream bais = new ByteArrayInputStream(inBytes);
 
-           ShaderDesc shaderDesc = compile(bais, shaderType, null, args[1], cmd.getOptionValue("platform", ""), cmd.getOptionValue("variant", "").equals("debug") ? true : false, false);
+           ShaderDesc shaderDesc = compile(bais, shaderType, null, args[1], cmd.getOptionValue("platform", ""), cmd.getOptionValue("variant", "").equals("debug") ? true : false, false, false);
```

Comment:
cmd-line compilation will never build spirv now, do we want that as an option?

Reply:
That's mostly for debugging/developing the ShaderProgramBuilder, right?
We can add it later when/if we need it.

Reply:
Yeah true.